packages/camera/camera/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.10.6
+
+* Fixes camera resource handling
```

Comment:
This could be more descriptive. I would use a sentence to explain the error that is being fixed. This PR also seems to fix 2 bugs, so I would include a sentence for each.